caluma/caluma_core/health_checks.py

```diff
@@ -0,0 +1,140 @@
+import uuid
+from io import BytesIO, StringIO
+
+from django.conf import settings
+from django.core import management
+from watchman.decorators import check
+
+from caluma.caluma_form import storage_clients
+from caluma.caluma_logging.models import AccessLog
+
+
+@check
+def _check_pending_migrations(db):
```

Comment:
I'd prefer having a different name for this argument (e.g. `db_name`) in order to not confuse it for the `db` fixture.